Mathlib/Topology/ContinuousFunction/ContinuousMapZero.lean

```diff
@@ -196,6 +197,23 @@ lemma coe_toContinuousMapHom [StarRing R] [ContinuousStar R] :
     ⇑(toContinuousMapHom (X := X) (R := R)) = (↑) :=
   rfl
 
+/-- The coercion `C(X, R)₀ → C(X, R)` bundled as a continuous linear map. -/
+@[simps]
+def toContinuousMapCLM (M : Type*) [Semiring M] [Module M R] [ContinuousConstSMul M R] :
+    C(X, R)₀ →L[M] C(X, R) where
+  toFun f := f
+  map_add' _ _ := rfl
+  map_smul' _ _ := rfl
+
+/-- The evaluation at a point, as a continuous linear map from `C(X, R)₀` to `R`. -/
+def evalCLM (𝕜 : Type*) {R : Type*} [CompactSpace X] [NormedField 𝕜] [NormedCommRing R]
+    [NormedSpace 𝕜 R] (x : X) : C(X, R)₀ →L[𝕜] R :=
+  (ContinuousMap.evalCLM 𝕜 x).comp (toContinuousMapCLM 𝕜 : C(X, R)₀ →L[𝕜] C(X, R))
+
+@[simp]
+lemma evalCLM_apply {𝕜 : Type*} {R : Type*} [CompactSpace X] [NormedField 𝕜] [NormedCommRing R]
```

Comment:
I guess `@[simps]` doesn't generate the right lemma here?

bors d+

Reply:
I tried, it doesn't unfold enough.